formats/base_links.json

```diff
@@ -7,6 +7,10 @@
       "description": "Prototype-stage taxonomy label for this content item",
       "$ref": "#/definitions/guid_list"
     },
+    "ordered_related_items": {
+      "description": "Related items. Mostly used for mainstream content to power the sidebar. Ordering of the links is determined by the editor in Panopticon.",
```

Comment:
This should say what "related items" actually are. Can this be anything with a content id?